### PR TITLE
New serverless pattern - EventBridge-SQS-ECS

### PR DESCRIPTION
**Pull request by Kiranpreet**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eb_sqs_ecs_project/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# AWS Service 1 to AWS Service 2
```
Comment (mavi888) [inline eb_sqs_ecs_project/README.md:1]: Please fix!

File: eb_sqs_ecs_project/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+This pattern << explain usage >>
```
Comment (mavi888) [inline eb_sqs_ecs_project/README.md:3]: Please complete

**Comment by mavi888** [comment]
Thanks for submitting the pattern. But this pattern cannot be merged.

- Please fix the README.md as it just the basic one from the template.
- Also follow the pattern for the directory structure as other patterns. 
For this one it would be instead of eb_sqs_ecs_project --> eventbridge-sqs-ecs-cdk

When you fix this i will continue reviewing the pattern itself, as now there are no instructions to follow.

Cheers,

**Comment by Kiranpreet** [comment]
Hi, 

Thanks for your comments. I've updated the project. Please let me know if any other changes are required. 

Thank you.

File: eb_sqs_ecs_project/README.md (hunk 1 above)
Comment (Kiranpreet) [reply eb_sqs_ecs_project/README.md:1]: Thank you. I've made the changes. Please review and let me know if any more updates are required.

File: eb_sqs_ecs_project/README.md (hunk 2 above)
Comment (Kiranpreet) [reply eb_sqs_ecs_project/README.md:3]: Done. Please review. Thank you.

**Comment by mavi888** [comment]
This pattern is not completed. You are still missing the README.md and pattern.json.

Please review the instructions on how to submit a pattern and fix this PR. https://github.com/aws-samples/serverless-patterns/issues/new?assignees=jbesw&labels=&template=new-serverless-pattern-submission.md&title=New+pattern+submission

**Comment by Kiranpreet** [comment]
Its done now. Added the required files. 

Thank you.

File: eventbridge-sqs-ecs-cdk/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    ```
+    cd eventbridge-sqs-ecs-cdk/src
+    ```
+3. Install the project dependencies
+    ```
+    python -m pip install -r python-app/requirements.txt
+    ```
+4. Deploy the stack to your default AWS account and region.
+    ```
+    cdk deploy
```
Comment (mavi888) [inline eventbridge-sqs-ecs-cdk/README.md:35]: This is a node cdk project, so the instructions are missing how to install the node dependencies, eg npm i before the cdk deploy. 
Also when trying to deploy this there are TS errors. Please fix the project and the instructions to for a person that downloads this project from GithHub.

File: eventbridge-sqs-ecs-cdk/example-pattern.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Trigger ECS task from SQS using EventBridge",
+  "description": "Create an application to create ECS tasks based on SQS messages pushed through EventBridge.",
+  "language": "Python",
```
Comment (mavi888) [inline eventbridge-sqs-ecs-cdk/example-pattern.json:4]: Language is Typescript not python as the infra is defined in Typescript

**Review by mavi888 — COMMENTED** [review]
Thanks for your submition, but this PR has some issues that need to be fixed before it can be merged.

Cheers,

File: eventbridge-sqs-ecs-cdk/example-pattern.json (hunk 4 above)
Comment (Kiranpreet) [reply eventbridge-sqs-ecs-cdk/example-pattern.json]: Oh, my bad. I have updated the file now.

File: eventbridge-sqs-ecs-cdk/README.md (hunk 3 above)
Comment (Kiranpreet) [reply eventbridge-sqs-ecs-cdk/README.md:39]: Hi, I have updated the file to include npm install command. However, I don't see any TS error at my end. Can you please provide more details on it?

Thank you.

**Comment by mavi888** [comment]
Thanks a lot for fixing the issues. 

Few more things before this is ready to get merged:
- the *d.ts and *.js files are not needed as those are generated later, please remove them from the project
- in the Readme in the Requirements please list Docker installed and running if not people will find some errors when deploying the project
- In the readme.md the intro is confusing, as it says that when a new event is sent to EB, it creates an ECS cluster, the cluster is already created. Please be more specific about what is happening when a new message is put in the queue. 
- Also in the readme and in the pattern.json it says that based on the number of messages, what you mean by that? How many messages? 
- In the testing part can you please put an example of an event.json file that matches the one that will trigger the rule, so it is easier to test and there is less guessing. 
- In the testing phase it says, "If successful, you'll see a new eventbridge bus" --> what you mean by successful? Putting the message or deploying the infra, as is written now, it seems that when you put the message and everything goes well, a new bus is created.

**Comment by Kiranpreet** [comment]
Thank you for providing your feedback. I've made the changes as suggested. Please let me know if you need any further updates.

**Pull request merged**